File: JavaScript/1041.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
const coord = input.split(" ");

// URI 1041 | COORDENADAS DE UM PONTO

let X = parseFloat(coord.shift());
let Y = parseFloat(coord.shift());

if (Y === 0 && X === 0) {
	console.log("Origem");
} else if (Y === 0) {
	console.log("Eixo X");
} else if (X === 0) {
	console.log("Eixo Y");
} else if (X > 0 && Y > 0) {
	console.log("Q1");
} else if (X > 0 && Y < 0) {
	console.log("Q4");
} else if (X < 0 && Y > 0) {
	console.log("Q2");
} else {
	console.log("Q3");
}

File: Python/1041.py
# BEE 1041 | COORDENADAS DE UM PONTO

coords = input().split(' ')
X = float(coords[0])
Y = float(coords[1])

if Y == 0 and X == 0:
    print("Origem")
elif Y == 0:
    print("Eixo X")
elif X == 0:
    print("Eixo Y")
elif X > 0 and Y > 0:
    print("Q1")
elif X > 0 > Y:
    print("Q4")
elif X < 0 and Y > 0:
    print("Q2")
else:
    print("Q3")
